src/estimators/bloom_filters.py

```diff
@@ -143,6 +143,33 @@ def __init__(self, length, random_seed=None):
         random_seed)
 
 
+class GeometricBloomFilter(AnyDistributionBloomFilter):
+  """Implement a Geometric Bloom Filter."""
+
+  @classmethod
+  def get_sketch_factory(cls, length, num_hashes=1):
+
+    def f(random_seed):
+      return cls(length, num_hashes, random_seed)
+
+    return f
+
+  def __init__(self, length, num_hashes=1, random_seed=None, probability=0.08):
+    """Creates a BloomFilter.
+
+    Args:
+       length: The length of bit vector for the bloom filter
+       random_seed: An optional integer specifying the random seed for
+         generating the random seeds for hash functions.
+    """
+    super().__init__(
+        any_sketch.SketchConfig([
+            any_sketch.IndexSpecification(
+                any_sketch.GeometricDistribution(length, probability), "geometric")
+        ], num_hashes=1, value_functions=[any_sketch.BitwiseOrFunction()]),
```

Comment:
Will you support num_hashes > 1? If so, num_hashes = num_hashes. otherwise, do you need num_hashes as arg?